common/roxiecommlib/roxiecommunicationclient.cpp

```diff
@@ -306,7 +306,7 @@ class CRoxieCommunicationClient: implements IRoxieCommunicationClient, public CI
             StringBuffer err;
             err.appendf("%d ", errCode);
             e->errorMessage(err);
-            DBGLOG("ERROR loading query info directly to roxie %s", err.str());
+            OERRLOG("ERROR loading query info directly to roxie %s", err.str());
```

Comment:
Probably ERRLOG (comment above indicates it is not fatal, so operator should not need to know)